tasks/main.yml

```diff
@@ -1,18 +1,22 @@
 ---
-# tasks file for cyhy_archive
```

Comment:
@jsf9k Should this comment have been removed?  I can create a PR for it if not, just wanted to verify

Reply:
Wait, I _did_ remove that comment in this PR!  I think I may be misunderstanding your question, @jmorrowomni.

Reply:
Sorry, yes, wasn't sure if it was supposed to be removed or not.  Based on your comment on another PR I realized it was intentional